integration_tests/e2e/induction/updateCheckYourAnswersChangeLinks.cy.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Cypress tests that test the Change links on the Check Your Answers page when updating an Induction
+ */
+import Page from '../../pages/page'
+import CheckYourAnswersPage from '../../pages/induction/CheckYourAnswersPage'
+import HopingToGetWorkValue from '../../../server/enums/hopingToGetWorkValue'
+import ReasonNotToGetWorkValue from '../../../server/enums/reasonNotToGetWorkValue'
+import InPrisonWorkValue from '../../../server/enums/inPrisonWorkValue'
+import InPrisonTrainingValue from '../../../server/enums/inPrisonTrainingValue'
+import AdditionalTrainingValue from '../../../server/enums/additionalTrainingValue'
+
+context(`Change links on the Check Your Answers page when updating an Induction`, () => {
+  const prisonNumber = 'G6115VJ'
+
+  beforeEach(() => {
+    cy.signInAsUserWithEditAuthorityToArriveOnPrisonerListPage()
```

Comment:
I've written some custom Cypress commands (`signInAsUserWithEditAuthorityToArriveOnPrisonerListPage`, `updateShortQuestionSetInductionToArriveOnCheckYourAnswers` and `updateLongQuestionSetInductionToArriveOnCheckYourAnswers`) to reduce duplication and simplify the readability of the tests.

The implementation of the new commands is down in `integration_tests/support/commands.ts` 👇